### DISCUSSION TITLE
Is Rod a viable solution for worker pools?

### DISCUSSION THREAD

**Discussion by khaelou** [go-rod/rod#783] [General]
Just curious, would a worker-pool of multiple workers executing Rod operate as expected? (3 workers browsing different sites, completing different tasks) My guess is no due to the speed of execution and the nature of driving the browser.

**Comment by ysmood** [comment]
Rod is designed to be concurrent. Search "page pool" in this project, you should find related info. The unit tests of Rod itself is concurrent.
